docs/source/release_notes/release-notes.rst

```diff
@@ -50,6 +50,11 @@ develop
     *    - Type
          - Change
 
+    *    - |logs|
+         - Messages to the `slow-lock-log` now log at `WARN` rather than `INFO`.
+           Increased logging level of messages to the `slow-lock-log` from `INFO` to `WARN`, these messages can indicate a problem so we should be sure they are visible.
```

Comment:
nit: why are we saying the same thing twice here?

Reply:
Errr... because all the other release note messages had 2 lines so I felt compelled to write 2 too.  I'll clean up, push and merge.